galaxy_importer/collection.py

```diff
@@ -41,7 +42,7 @@
     namedtuple("CollectionFilename", ["namespace", "name", "version"])
 
 
-def import_collection(file, filename=None, logger=None, cfg=None):
+def import_collection(file=None, filepath=None, filename=None, logger=None, cfg=None):
```

Comment:
Feels odd having all the args be optional. Plus 'file', 'filepath', 'filename'. I'm guessing this is to maintain API compat since this is the main "public" API?